### PR TITLE
adjust form elements typos

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

In reference to unresolved comments in https://github.com/camunda/camunda-platform-docs/pull/1570, this PR cleans up any lingering typos following its merge.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: forms-element-library-textarea
 title: Text area
-description: Learn about the library textarea form element to read and edit multiline textual data.
+description: Learn about the textarea form element to read and edit multiline textual data.
 ---
 
 A text area allowing the user to read and edit multiline textual data.
```
Comment (akeller) [inline docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md:7]: Is it "textarea" after the HTML element `<textarea>`? I see a discrepancy between the description and the first line.

Comment (akeller) [reply docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md:7]: textarea vs. text area

Comment (christinaausley) [reply docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md:7]: @Skaiir @pinussilvestrus do you have thoughts here? How is this referenced in the UI? Happy to clarify and add into the glossary.

Comment (pinussilvestrus) [reply docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md:7]: It's indeed a `textarea` HTML element in the end, but we call it `text area` in the UI.

Comment (pinussilvestrus) [reply docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md:7]: The docs should be consistent with the UI here I think.

Comment (christinaausley) [reply docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md]: Updated the description 👍

**Review by akeller — APPROVED** [review]
Thank you!

**Pull request merged**